src/test/resources/for-execution/unions.golo

```diff
@@ -178,6 +178,37 @@ function test_singleton = {
   }
 }
 
+function test_match_methods = {
```

Comment:
Which makes me thing that we'd benefit from having a cool Golo-esque testing module.